raiden/tests/conftest.py

```diff
@@ -131,7 +131,7 @@ def check_geth_version_for_tests(blockchain_type):
     ).communicate()
     supported, _, our_version = is_supported_client(geth_version_string.decode())
     if not supported:
-        sys.exit(
```

Comment:
huh, I never know that `sys.exit` accepted a str.

Reply:
I actually asked myself every time I looked at this how come this works. And every time I have to check it for myself :smile: